spec/components/all_components_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe "AllComponents", type: :controller do
+  Dir.glob("app/views/components/*.erb").each do |filename|
+    template = filename.split("/").last
+    component_name = template.sub("_", "").sub(".html", "").sub(".erb", "").gsub("-", "_")
+
+    describe(component_name) do
+      yaml_file = "#{__dir__}/../../app/views/components/docs/#{component_name}.yml"
+
+      it "is documented" do
+        expect(File).to exist(yaml_file)
+      end
+
+      it "has the correct documentation" do
+        yaml = YAML.unsafe_load_file(yaml_file)
```

Comment:
Strong suspicion it's because whoever wrote the code just didn't know about permitting classes during a safe load (if you just change it to safe load it will fail, but if you change it to safe load and permit Time objects, it's okay. I've done that now)

Reply:
(Marking as resolved)